Source/HarmonyPatches/HPatcher.cs

```diff
@@ -39,31 +39,6 @@ public static void Init()
                 Log.Error($"PrisonLaborException: failed to proceed harmony patches: {e.InnerException.Message}");
                 Log.Error(e.ToString());
             }
-
-            // SECTION - Patches with references in method
-            try
-            {
-                harmony.Patch(
-                    typeof(Pawn_CarryTracker).GetMethod("TryDropCarriedThing",
-                        new[]
-                        {
-                        typeof(IntVec3), typeof(ThingPlaceMode), typeof(Thing).MakeByRefType(),
-                        typeof(Action<Thing, int>)
-                        }),
-                    new HarmonyMethod(null), new HarmonyMethod(typeof(Patches_PermissionFix.ForibiddenDropPatch).GetMethod("Postfix")));
-                harmony.Patch(
-                    typeof(Pawn_CarryTracker).GetMethod("TryDropCarriedThing",
-                        new[]
-                        {
-                        typeof(IntVec3), typeof(int), typeof(ThingPlaceMode), typeof(Thing).MakeByRefType(),
-                        typeof(Action<Thing, int>)
-                        }),
-                    new HarmonyMethod(null), new HarmonyMethod(typeof(Patches_PermissionFix.ForibiddenDropPatch).GetMethod("Postfix2")));
-            }
-            catch (Exception e)
-            {
-                Log.Error($"PrisonLaborException: failed to proceed harmony patches (reference section): {e.InnerException.Message}");
-            }
```

Comment:
I used the appropriate HarmonyAttributes for Patching instead, which is a bit cleaner in my opinion. Also this wasn't working anymore since the arguments got changed from 'ref' to 'out'

Reply:
When I was writing this back in the past I couldn't make it by attribute